src/components/Text/Text.module.css

```diff
@@ -5,8 +5,78 @@
 \*------------------------------------*/
 
 /**
- * 1) 
+ * 1) h1, h2, h3, h4, h5, h6
+ * 2) Styles are either controlled by as or size declaration
  */
-/* stylelint-disable-next-line block-no-empty */
-.text {
+.text,
+.text--body, 
+.text--md {
+  @mixin reset;
+  
+  @mixin eds-theme-typography-body-text;
+}
+
+/**
+ * Sizes
+ */
+.text--lg {
+  @mixin eds-theme-typography-body-text-lg;
+}
+.text--sm {
+  @mixin eds-theme-typography-body-text-sm;
+}
+.text--xs {
+  @mixin eds-theme-typography-body-text-xs;
+}
+.text--caption {
+  @mixin eds-theme-typography-caption-text-md;
+}
+.text--overline {
+  @mixin eds-theme-typography-kicker;
+}
+
+/**
+ * Color
+ */
+.text--alert {
+  color: var(--eds-theme-color-text-utility-error);
+}
+.text--base {
+  color: var(--eds-theme-color-text-neutral-default);
+}
+.text--brand {
+  /* TODO: replace with tier 2 token */
+  color: var(--eds-color-brand-grape-700);
+}
+.text--info {
+  /* TODO: doesn't exist, replace with tier 2 token */
+  color: var(--eds-color-other-blue-700);
```

Comment:
actually I think the blue tokens fully no longer exist, see: https://www.figma.com/file/IfMWAtWqNrQprImlco4pi0/EDS---UI-Foundations?node-id=58%3A4827

we should ask designers what to do about deprecating the `info` variants